cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/elements/hvdc/IslandEndHvdc.java

```diff
@@ -32,7 +29,7 @@
     // T2: two transformers, C2: two acDcConverters, LS2: two dcLineSegments
     // TN: n transformers (usually 2), CN: n acDcConverters (usually 2), LSN: n dcLineSegments (usually 2)
     enum HvdcEndType {
-        HVDC_T0_C1_LS1, HVDC_T1_C1_LS1, HVDC_TN_CN_LSN, HVDC_T1_C1_LS2, HVDC_T2_C2_LS1,
+        HVDC_T0_C1_LS1, HVDC_T0_C1_LS2, HVDC_T0_C2_LS1, HVDC_T1_C1_LS1, HVDC_TN_CN_LSN, HVDC_T1_C1_LS2, HVDC_T2_C2_LS1,
```

Comment:
Through that PR, you have found to new end type that were not supported before, why? Or it is just to be more generic?

Reply:
Before the DC configurations were only analyzed at bus branch level. Now we analyze them at both levels, node breaker level and bus branch level. All the real IOPs with DC components are defined at the node breaker level and at this level some of the configurations are a bit different. We support the same configurations, but at both levels.